app.py

```diff
@@ -1,36 +1,48 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import trello_items as item_store
+
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
I don't think you need this line as `flask run` will load the `.env` file automatically for you

Reply:
Removed